app/controllers/camp.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+class camp extends Controller {
+
+    function __construct() {
+        $this->load_model('camp_model');
+    }
+
+    function index() {
+        $this->load_view('camp/index');
+    }
+
+    function add_new() {
```

Comment:
This PR is just about camp list. This is not related to this PR. Please remove it?